### ISSUE TITLE
Updating Codiad using the auto update feature deletes any custom themes 

### ISSUE THREAD

**Issue by jamieisboss** [Codiad/Codiad#810]
I had been working on a hidpi version of the default theme, updated... gone.

I'd installed the other themes from the marketplace to get a feel for how they worked, they are gone too. if anyone knows any good way of recovering the files that'd be cool, spent quite a while on it. It's on a vps so there are no devices in dev or media or mount.

:sob:

**Comment by daeks** [comment]
Which autoupdate do you mean? Updating with git? [my thirdparty plugin](https://github.com/daeks/Codiad-AutoUpdate)? 
But generelly spoken, no there is no way to recover files as long you or your vps host has a backup

**Comment by jamieisboss** [comment]
I used the [plugin you mention](https://github.com/daeks/Codiad-AutoUpdate), I assume it should leave the themes directory alone like the workspace and the plugins ones but gedanken-bug because you want to keep the main theme updated with the codiad version because you're adding features and they need to be styled by default.

Unfortunately only my workspace directory is in my backup and my VPS host gives you some ftp space and leaves the backing up to you. Didn't think at the time. I've tried installing extundelete and played with that for an while but it's gone i reckon. 

It was mostly css changes anyway It's all still in my brain It won't take nearly as long to recreate. No worries, the few times I've lost chunks of code, it's always turned out much better the second time anyway. :sunglasses:
